cmd/operator/main.go

```diff
@@ -114,6 +127,10 @@ func main() {
 		setupLog.Error(err, "unable to start manager")
 		os.Exit(1)
 	}
+	if validateManagedClusterCRDExists(mgr) != nil {
```

Comment:
@rccrdpccl FYI this will be another thing you'll need to install for sylva deployments when not using MCE.